client/client-multi/client-c/src/platform/system_logger.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * system_logger.h
+ *
+ *  Created on: Dec 31, 2014
+ *      Author: avp
+ */
+
+#ifndef SRC_KAA_UTILITIES_SYSTEM_LOGGER_H_
```

Comment:
By code convention, guard ifdefs should be identical to the upper case file name with "_H_" in the end. Please change here and everywhere.